drivers/storage/ec2/ec2.go

```diff
@@ -409,61 +446,60 @@ func (d *driver) CreateVolume(
 		return nil, err
 	}
 
-	volumes, err = d.GetVolume(resp.VolumeId, "")
+	volumes, err = d.GetVolume(*resp.VolumeId, "")
 	if err != nil {
 		return nil, err
 	}
 
 	// log.Println(fmt.Sprintf("Created volume: %+v", volumes[0]))
 	return volumes[0], nil
-
 }
 
 func (d *driver) createVolume(
 	runAsync bool, volumeName, volumeID, snapshotID, volumeType string,
 	IOPS, size int64,
-	availabilityZone string) (*ec2.CreateVolumeResp, error) {
+	availabilityZone string) (*ec2.Volume, error) {
 
 	if volumeID != "" && runAsync {
-		return &ec2.CreateVolumeResp{}, errors.ErrRunAsyncFromVolume
+		return &ec2.Volume{}, errors.ErrRunAsyncFromVolume
 	}
 
 	var err error
 
 	var server ec2.Instance
 	if server, err = d.getInstance(); err != nil {
-		return &ec2.CreateVolumeResp{}, err
+		return &ec2.Volume{}, err
 	}
 
 	if volumeID != "" {
 		if snapshotID, err = d.createVolumeCreateSnapshot(
 			volumeID, snapshotID); err != nil {
-			return &ec2.CreateVolumeResp{}, err
+			return &ec2.Volume{}, err
 		}
 	}
 
 	d.createVolumeEnsureAvailabilityZone(&availabilityZone, &server)
 
-	options := &ec2.CreateVolume{
-		Size:       size,
-		SnapshotId: snapshotID,
-		AvailZone:  availabilityZone,
-		VolumeType: volumeType,
-		IOPS:       IOPS,
+	options := &ec2.CreateVolumeInput{
+		Size:             &size,
+		SnapshotId:       &snapshotID,
+		AvailabilityZone: &availabilityZone,
+		VolumeType:       &volumeType,
+		Iops:             &IOPS,
```

Comment:
we've confirmed that this is a problem; the "zero-value" of the IOPS parameter gets passed in here and results in operation errors when volumetype standard is used